Entity/LedgerEntryRepository.php

```diff
@@ -14,6 +14,7 @@
 use Mautic\CoreBundle\Entity\CommonRepository;
 use Mautic\LeadBundle\Entity\Lead;
 
+const MAUTIC_CONVERSION_STATUS = 'sell';
```

Comment:
Constant should be in a class, not global. Also, we went with 'converted'.

Reply:
done